### PR TITLE
support puppet 4.0.0 and puppetserver AIO

### PR DESCRIPTION
**Pull request by jlambert121**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -5,16 +5,22 @@
 class puppet_hipchat::params {
 
   $package_name   = 'hipchat'
-  $install_hc_gem = true
   $puppetboard    = false
   $dashboard      = false
 
   if str2bool($::is_pe) {
+    $install_hc_gem = true
```
Comment (jamtur01) [inline manifests/params.pp]: Can you align the =

File: manifests/params.pp [hunk 2]
```diff
@@ -5,16 +5,22 @@
...
     $provider        = 'pe_gem'
     $owner           = 'pe-puppet'
     $group           = 'pe-puppet'
+  } elsif versioncmp('4.0.0', $::puppetversion) < 1 {
+    $puppetconf_path = '/etc/puppetlabs/puppet'
+    $install_hc_gem  = false
+    $owner           = 'puppet'
+    $group           = 'puppet'
   } else {
+    $install_hc_gem = true
```
Comment (jamtur01) [inline manifests/params.pp]: Ditto

**Comment by jamtur01** [comment]
Thanks!

**Comment by jlambert121** [comment]
Fixed - sorry about that.

**Pull request merged**